src/common/constants/constants.ts

```diff
@@ -241,3 +241,11 @@ export const TelemetryEvents = {
     applicationStart: "Application start",
     disableTelemetry: "Disable telemetry",
 };
+
+export const MaxDurations = {
```

Comment:
Might be best to make each of these its own variable, then do Hours = Days *24 to make it clear thats where its coming from